script/phase-1b/13-Nst$PlaceholderPreFarmingCheck.s.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {Script} from "forge-std/Script.sol";
+import {Reader} from "../helpers/Reader.sol";
+
+contract Phase1b_Nst$PlaceholderPreFarmingCheckScript is Script {
+    ChainlogLike internal constant chainlog = ChainlogLike(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+
+    function run() external returns (bool) {
+        Reader deps = new Reader("");
+        deps.loadDependenciesOrConfig();
+
+        address admin = chainlog.getAddress("MCD_PAUSE_PROXY");
+        address nst = deps.envOrReadAddress("FOUNDRY_NST", ".nst");
+        address rewards = deps.readAddress(".rewards");
+
+        require(StakingRewardsLike(rewards).owner() == admin, "PlaceholderPreFarmingCheck/admin-not-owner");
+        require(
+            StakingRewardsLike(rewards).rewardsToken() == address(0),
+            "PlaceholderPreFarmingCheck/invalid-rewards-token"
+        );
```

Comment:
`rewardsToken` is not editable on `StakingRewards`. Changing it requires a redeploy (or updating the contract so it's possible).

Reply:
That's by design. We don't wand a reward token for the pre-farm.